features/logging/cluster_log_forwarder.feature

```diff
@@ -495,3 +495,62 @@ Feature: cluster log forwarder features
     And evaluation of `JSON.parse(@result[:stdout])['count']` is stored in the :app_log_count_2 clipboard
     And the expression should be true> cb.app_log_count_2 > 0
     """
+  # @author kbharti@redhat.com
+  # @case_id OCP-39786
+  @admin
+  @destructive
+  Scenario: Send logs to both external fluentd and internalES
+    #Creating secure fluentd receiver
+    Given I switch to cluster admin pseudo user
+    Given fluentd receiver is deployed as secure with server_auth_share enabled in the "openshift-logging" project
+    # Creating app
+    Given I switch to the first user
+    And I create a project with non-leading digit name
+    And evaluation of `project` is stored in the :proj clipboard
+    And I obtain test data file "logging/loggen/container_json_log_template.json"
+    When I run the :new_app client command with:
+      | file | container_json_log_template.json |
+    Then the step should succeed
+    #Creating secure secret and ConfigMap
+    Given I switch to cluster admin pseudo user
+    Given admin ensures "secure-forward" config_map is deleted from the "openshift-logging" project after scenario
+    Given admin ensures "secure-forward" secret is deleted from the "openshift-logging" project after scenario
+    Given I run the :create_secret client command with:
+      | name         | secure-forward           |
+      | secret_type  | generic                  |
+      | from_file    | ca-bundle.crt=ca.crt     |
+      | n            | openshift-logging        |
+    Then the step should succeed
+    Given I obtain test data file "logging/clusterlogforwarder/fluentd/secure/secure-forward-cm.yaml"
+    Given I run the :create client command with:
+      | f | secure-forward-cm.yaml |
+    Then the step should succeed
+    And I wait for the "secure-forward" config_map to appear
+    # Creating Cluster Logging Instance
+    Given I obtain test data file "logging/clusterlogging/example_indexmanagement.yaml"
+    When I create clusterlogging instance with:
+      | remove_logging_pods | true                         |
+      | crd_yaml            | example_indexmanagement.yaml |
+    Then the step should succeed
+    Given I wait for the "fluentd" daemon_set to appear up to 300 seconds
+    And <%= daemon_set('fluentd').replica_counters[:desired] %> pods become ready with labels:
+      | logging-infra=fluentd |
+    # Check logs in fluentd receiver
+    Given I use the "openshift-logging" project
```

Comment:
I think we can remove these steps here because in step `I create clusterlogging instance with` we already switch to openshift-logging project and it checks the EFK status after creating cl/instance.

Reply:
The step has been removed.